tests/unit/views/test_model_view.py

```diff
@@ -41,41 +41,33 @@ class ItemView(RetrieveUpdateDestroyModelView):
 
 
 @pytest.fixture
-def database():
-    db = SqliteDatabase(':memory:')
-    db.bind(MODELS)
-    db.connect()
-    db.create_tables(MODELS)
+def database(database_maker):
+    return database_maker(models=MODELS)
```

Comment:
Do we need this `database_maker` to be a fixture?

Reply:
Technically speaking, we don't *need* anything to be fixture-decorated but since that's the recommended `pytest` way, why not following it?

Reply:
I didn't know `pytest` recommends this way. I actually don't see why we should have a fixture if it's not used as a fixture (for tests).

Reply:
On a second (third actually) thought, I think it's best that we keep it as a fixture. Reasons are:

1. Its friend, `http_client_maker()` follows the same pattern and it *must* be a fixture so that it receives the` aiohttp_client` fixture injection.
2. We shouldn't force testers to always create an intermediate fixture so that they can use `database_maker`. It should be usable directly as a fixture itself, being invoked as a db factory inside the test.